packages/web-components/src/components/va-official-gov-banner/va-official-gov-banner.scss

```diff
@@ -0,0 +1,304 @@
+  flex: 0 1 auto;
+  width: auto;
+  max-width: 100%;
+}
+
+.grid-row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: -0.75rem;
+  margin-right: -0.75rem;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.col {
+  align-items: flex-start;
+  display: flex;
+  flex: 0 0 46%;
+  padding-left: 0.75rem;
+  padding-right: 0.75rem;
+}
+
+@media (max-width: 768px) {
+  .col {
+    flex: unset;
+  }
+}
+
+header {
+  padding-bottom: 6px;
+  padding-top: 6px;
+  position: relative;
+}
+
+header.expanded .header-action {
+  display: none;
+}
+
+@media (max-width: 768px) {
+  header {
+    min-height: 32px;
+  }
+
+  header.expanded {
+    padding-right: 56px;
+  }
+}
+
+.inner {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1024px;
+  padding-left: 16px;
+  padding-right: 16px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  padding-right: 0;
+}
+
+.header-flag {
+  float: left;
+  margin-right: 8px;
+  width: 16px;
+}
+
+.header-text {
+  margin-bottom: 2px;
+  margin-right: 5px;
+  margin-top: 2px;
+  line-height: 1.1;
+}
+
+.header-action {
+  color: #005ea2;
+  line-height: 1.1;
+  margin-bottom: 0;
+  margin-top: 2px;
+  text-decoration: underline;
+}
+
+@media (min-width: 768px) {
+  .header-action {
+    display: none;
+  }
+}
+
+button:not([disabled]):focus {
+  outline: 2px solid var(--color-gold-light);
+}
+
+button[aria-expanded=false], button[aria-expanded=true] {
+  background-image: none;
+}
+
+@media (max-width: 768px) {
+  button {
+    width: 100%;
+  }
+}
+
+button {
+  border: 0;
+  color: #005ea2;
+  font-size: 12px;
+  margin: 0;
+  margin-left: 0.5rem;
+  padding-left: 0;
+  padding-right: 0;
+  position: relative;
+}
+
+@media (max-width: 768px) {
+  button {
+    -webkit-font-smoothing: inherit;
+    background-color: transparent;
+    border: 0;
+    border-radius: 0;
+    box-shadow: none;
+    margin: 0;
+    text-align: left;
+    left: 0;
+    position: absolute;
+    bottom: 0;
+    top: 0;
+    color: #005ea2;
+    display: block;
+    height: auto;
+    line-height: 1.1;
+    padding: 0;
+    text-decoration: none;
+    width: 100%;
+  }
+}
+
+button:hover {
+  color: #1a4480;
+  cursor: pointer;
+}
+
+.button-text {
+  position: absolute;
+  left: -999em;
+  right: auto;
+  text-decoration: underline;
+}
+
+@media (min-width: 768px) {
+  .button-text {
+    position: static;
+    display: inline;
+  }
+}
+
+// Mobile display dropdown/expand more arrow.
+@media (max-width: 768px) {
+  header .header-action::after, header.expanded .header-action::after {
+    background-color: #005ea2;
+  }
+
+  header.expanded .header-action::after, header .header-action::after {
+    background: 0 0;
+    background-color: #005ea2;
+    -webkit-mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_more.svg") no-repeat center/contain;
+    mask: url("../../../../../node_modules/@uswds/uswds/dist/img/usa-icons/expand_more.svg") no-repeat center/contain;
```

Comment:
Similar to the hardcoded hex values, this direct link to the v3 icons will no longer be necessary once this component has been converted to use the v3 modules (ie. @use "usa-banner").